### ISSUE TITLE
white bar on Firefox/macOS when scrolling up using trackpad

### ISSUE THREAD

**Issue by swedgwood** [matrix-org/matrix.org#1948]
Minor visual bug on the website, which only happens when I scroll up using the trackpad.

On Firefox 115.0.2 / macOS Ventura 13.4.1

https://github.com/matrix-org/matrix.org/assets/28223854/ddf1acaa-11c8-410d-8f46-48fe7e44f993

I don't have Chrome installed for testing, but in Safari the top bar moves down with the rest of the page (and the gap above the is filled in with black, rather than white)

**Comment by thibaultamartin** [comment]
Indeed, that is because we use `position: fixed;` for the header instead of `position: sticky`!

**Comment by Drugjudy** [comment]
> Indeed, that is because we use `position: fixed;` for the header instead of `position: sticky`!

can i change it to sticky? but then there is a white gap in between so ? let it be?
